jbehave-support-core/src/main/java/org/jbehavesupport/core/verification/VerifierResolver.java

```diff
@@ -16,4 +15,13 @@
      * @return Single found verifier.
      */
     Verifier getVerifierByName(String name);
+
+    /**
+     * Finds single registered bean by verifier name, if empty, returns a default one. Exactly one must exist otherwise {@link NoUniqueBeanDefinitionException} or {@link NoSuchBeanDefinitionException}
+     *
+     * @param name Verifier name regards bean setup, or from {@link VerifierNames}
+     * @param defaultVerifierName Name of a default verifier
+     * @return Single found verifier or the specified defauld verifier
+     */
+    Verifier getVerifierByName(String name, String defaultVerifierName);
```

Comment:
personally, I don't like the contract of this new method, I can't remember that I'd see something similar anywhere else ... unfortunately, I don't have a better proposal ... I tend to the following code, which we have already there 
`verifierResolver.getVerifierByName(defaultIfEmpty(verifierName, EQ));` ... in any case, if others are happy with it, I'll shut up 😄